documentation/design-docs/dotnet-monitor.yaml

```diff
@@ -0,0 +1,457 @@
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "duration"
+        in: "query"
+        description: "Duration of the CPU trace"
+        required: false
+        type: "integer"
+        default: "30"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /cpuprofile/{uid}:
+    get:
+      tags:
+      - "traces"
+      operationId: "getcputrace"
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "uid"
+        in: "path"
+        description: "Process ID of the process to collect a process dump"
+        required: true
+        type: "string"
+      - name: "duration"
+        in: "query"
+        description: "Duration of the CPU trace"
+        required: false
+        type: "integer"
+        default: "30"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /trace/:
+    post:
+      tags:
+      - "traces"
+      operationId: "postdefaulttrace"
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "duration"
+        in: "query"
+        description: "Duration of the trace"
+        required: false
+        type: "integer"
+        default: "30"
+      - name: "configuration"
+        in: body
+        schema:
+          type: object
+          items:
+            $ref: "#/definitions/EventPipeConfiguration"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /trace/{uid}:
+    post:
+      tags:
+      - "traces"
+      operationId: "posttrace"
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "duration"
+        in: "query"
+        description: "Duration of the trace"
+        required: false
+        type: "integer"
+        default: "30"
+      - name: "uid"
+        in: "path"
+        description: "ID of the process to collect a process dump"
+        required: true
+        type: "string"
+      - name: "configuration"
+        in: body
+        schema:
+          type: object
+          items:
+            $ref: "#/definitions/EventPipeConfiguration"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /logs/:
+    get:
+      tags:
+      - "logs"
+      operationId: "getdefaultlogs"
+      produces:
+      # Newline-delimited JSON- http://www.ietf.org/rfc/rfc4288.txt
+      - "application/x-ndjson"
```

Comment:
As we discussed, this type does not appear to stream to the browser, but rather creates a file.

Reply:
Yeah. As far as I can tell `text/event-stream` is the only `Content-Type` that causes this behavior in the browser

Reply:
Does the order of the content types imply a default? I'd propose the default should be whatever shows up better in browser for someone doing it manually (text/event-stream it sounds like?). Anyone programatically hitting the API can specify the content type they want.